lib/beaker/subcommand.rb

```diff
@@ -49,55 +53,134 @@ def initialize(*args)
     class_option :'exclude-tags', :type => :string, :group => 'Beaker run'
     class_option :'xml-time-order', :type => :boolean, :group => 'Beaker run'
 
-    desc "init HYPERVISOR", "Initialises the beaker test environment configuration"
+    desc "init BEAKER_RUN_OPTIONS", "Initializes the required configuration for Beaker subcommand execution"
     option :help, :type => :boolean, :hide => true
     long_desc <<-LONGDESC
+      Initializes the required .beaker configuration folder. This folder contains
+      a subcommand_options.yaml file that is user-facing; altering this file will
+      alter the options subcommand execution.
 
-      Initialises a test environment configuration for a specific hypervisor
-
-      Supported hypervisors are: vagrant (default), vmpooler
-
+      Also modified in this operation is the .subcommand_state file that is used
+      by beaker to determine subcommand state. You as a user should never have to
+      edit this file.
     LONGDESC
```

Comment:
So I guess I don't like how this desc is telling the user what's going on under the hood, or how we're telling them about the .subcommand_state file, which they shouldn't really care about? I think a nicer desc would be what was there originally (I think it's important that we list out supported hypervisors at least), and then when we execute init, print out a message like "Written options to .beaker/subcommand_options.yaml". That way, we advertise the config location to the user, which they would probably be inclined to inspect themselves. I think we should hide any references to .subcommand_state as much as possible. I don't know if we should call these files subcommand_*, since I guess "subcommand" is too implementation-specific in that it relates to this particular phase of beaker development. I would be inclined to call them something like ".beaker_state" and "beaker_options", but I'm happy to go with whatever the majority decides 👍